app/views/application/previewapplication/generalDetailsViewSection.scala.html

```diff
@@ -1,5 +1,27 @@
 @(generalDetails: connectors.ExchangeObjects.GeneralDetailsExchange)(edit: Html)
 
+@import _root_.forms.FastPassForm
+
+
+@displayFastPassSection(id:String, display:Boolean, heading: String, text: => String) = @{
+    if(display){
+        <h3 class="heading-small">{heading}</h3>
+        <p id={id}>{text}</p>
+    }
+}
+
+@getYesOrNo(value:Boolean) = @{
+    if(value) "Yes" else "No"
+}
+
+@getFastPassTypeDescription(fastPassType: String) = @{
+    FastPassForm.FastPassTypes.find(_._1 == fastPassType).map(_._2).getOrElse("")
```

Comment:
Seq[(String, String)] is what most of the checkbox/radio widgets will take as input parameters. I am trying to use the same data structure here. But this particular code has now been simplified to 
FastPassForm.FastPassTypes.toMap.getOrElse(fastPassType, "")